### ISSUE TITLE
Association "add to" returns 404 on Ubuntu Server.

### ISSUE THREAD

**Issue by glemiere** [balderdashy/sails-docs#562]
Hi !
I followed instruction for adding associates datas here : http://sailsjs.org/documentation/reference/blueprint-api/add-to

Postman returns a 404 error not found, it's strange because populate where works perfectly :
http://sailsjs.org/documentation/reference/blueprint-api/populate-where

Is anybody has this problem too ?

**Comment by sailsbot** [comment]
Thanks for posting, @glemiere.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
